src/helpers/generateToken.js

```diff
@@ -0,0 +1,18 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import Cryptr from 'cryptr';
+
+dotenv.config();
+const secret = process.env.SECRET;
+const cryptr = new Cryptr(secret);
+/**
+ * @description This function generates and encrypts JWT token
+ * @param  {integer} time
+ * @param  {object} payload
+ * @returns {string} encrypted JWT token
+ */
+const generateToken = (time, payload) => {
```

Comment:
This is a good implementation 👍 . But I do feel we could spend less time using the crypto method we agreed